### ISSUE TITLE
feat: add emoji support to project icons

### ISSUE THREAD

**Issue by RealVishy** [johannesjo/super-productivity#3558]
## Problem Statement

When creating new projects, I'm annoyed there aren't enough icons to match with the projects I do. This means i have to pick icons that don't exactly match and aren't visually distinct.

## :grey_question: Possible Solution

Allow users to paste in emojis for project icons and/or provide an emoji picker so that users can paste in project icons that are more distinctive

## :arrow_heading_up: Describe alternatives you've considered

Increasing the amount of the project icons so that they make sense for more project use cases.

## :heavy_plus_sign: Additional context

https://github.com/johannesjo/super-productivity/issues/3463

A more general issue about adding icons through the whole app.

**Comment by stavby** [comment]
I can take this :)

**Comment by tsepanx** [comment]
But as I see in app, there is already possibility to set emoji as an icon (by pasting it)
![image](https://github.com/user-attachments/assets/f6235ecd-5526-405d-97d7-3909c212ca4b)
![image](https://github.com/user-attachments/assets/13d9367b-a052-4255-b432-91cd55875324)

**Comment by johannesjo** [comment]
You're right @tsepanx ! Maybe we should make it a little more accessible though.

**Comment by RealVishy** [comment]
We could just edit the text to say you can add one from your clipboard?

**Comment by johannesjo** [comment]
Emoji support will be improved with the next release! Closing this.
